komga-webui/src/views/Home.vue

```diff
@@ -163,14 +185,37 @@ export default Vue.extend({
     isAdmin (): boolean {
       return this.$store.getters.meAdmin
     },
+    themeItems (): any[] {
+      return Object.values(Theme).map(theme => (
+        {
+          text: theme,
+          value: theme,
+        }
+      ))
+    },
   },
   methods: {
     toggleDrawer () {
       this.drawerVisible = !this.drawerVisible
     },
-    toggleDarkMode () {
-      this.$vuetify.theme.dark = !this.$vuetify.theme.dark
-      this.$cookies.set(cookieDarkMode, this.$vuetify.theme.dark, Infinity)
+    setTheme (e: string) {
+      this.$cookies.set(cookieTheme, e)
+      this.changeTheme(e as Theme)
+    },
+    changeTheme (theme: Theme) {
```

Comment:
you can coerce those 2 functions in one, and use `theme.valueOf()` to get the string value for the cookie